<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResEditor</class>
 <widget class="QMainWindow" name="ResEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор ресурсов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="headerHidden">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string notr="true">-</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">-</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">-</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">-</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>654</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Информация</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="action_Qt"/>
    <addaction name="actionUpdate_history"/>
   </widget>
   <widget class="QMenu" name="menuLang">
    <property name="title">
     <string>Язык</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInfo"/>
   <addaction name="menuLang"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionUpdate_history">
   <property name="text">
    <string>История изменений</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Сохранить как ...</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>Про Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
